lib/local-storage-adapter.js

```diff
@@ -20,7 +19,7 @@ function createLocalStorageAdapter (queueName) {
   var setPending = false
   var queueCache = []
 
-  var flushOnDefer = _.debounce(flush, 0)
+  var flushOnDefer = onceOnDefer(flush)
```

Comment:
The old version would debounce, so if you call it multiple times it will only get called once. This would then reset after it is actually called. That is now the same that is happening here, I guess it is just way more explicit now.